AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/EventManagerInviteController.cs

```diff
@@ -0,0 +1,114 @@
+﻿using AllReady.Areas.Admin.Features.EventManagerInvites;
+using AllReady.Areas.Admin.ViewModels.ManagerInvite;
+using AllReady.Features.Events;
+using AllReady.Models;
+using AllReady.Security;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace AllReady.Areas.Admin.Controllers
+{
+    [Area("Admin")]
+    [Authorize(nameof(UserType.OrgAdmin))]
+    public class EventManagerInviteController : Controller
+    {
+        private readonly IMediator _mediator;
+        private readonly UserManager<ApplicationUser> _userManager;
+
+        public EventManagerInviteController(IMediator mediator, UserManager<ApplicationUser> userManager)
+        {
+            _mediator = mediator;
+            _userManager = userManager;
+        }
+
+        // GET: Admin/Invite/Send/5
+        [Route("[area]/[controller]/[action]/{eventId:int}")]
+        public async Task<IActionResult> Send(int eventId)
+        {
+            var @event = await _mediator.SendAsync(new EventByEventIdQuery { EventId = eventId });
```

Comment:
@mk0sojo I would add the OrganizationId to the ViewModel so you can make this check. The ViewModel is there to support the GET operation and the razor view(s) that consume it. Adding the OrganizationId to the ViewModel is ok even if the razor view is not using it as the controller needs to make this check.